docs/monitoring_metrics.py

```diff
@@ -0,0 +1,73 @@
+from operator import itemgetter
```

Comment:
Should this be in examples instead of docs root?

Reply:
@BenWilson2 This script generates the metrics table for Monitoring documentation. Is the `examples` for that?